### ISSUE TITLE
[Training Issue] Program was stuck while training a darknet yolov3 model

### ISSUE THREAD

**Issue by Rahn80643** [eric612/MobileNet-YOLO#205]
Model :  yolov3

Backbone : darknet53

Questions :
Hi, I trained a yolov3 model with backbone as darknet53, when the iteration is 1000, the program stuck at the message "Ignoring source layer Yolov3Loss1", "Ignoring source layer Yolov3Loss2", "Ignoring source layer Yolov3Loss3". 

The dataset I used is pascal voc, and I replace the upsample layers with deconvolution layers. I suspect if this occurs when testing the caffemodel, has anyone ever tried to train a darknet53-yolov3 model ?

Log files : 
[train-2019-11-11-11-04-08.log.txt](https://github.com/eric612/MobileNet-YOLO/files/3829778/train-2019-11-11-11-04-08.log.txt)

![20191111_stuck](https://user-images.githubusercontent.com/8951991/68562492-d7be7980-0484-11ea-88f8-68f5c189540c.png)

Other : 
[solver.prototxt.txt](https://github.com/eric612/MobileNet-YOLO/files/3829784/solver.prototxt.txt)
[yolov3-test.prototxt.txt](https://github.com/eric612/MobileNet-YOLO/files/3829785/yolov3-test.prototxt.txt)
[yolov3-train.prototxt.txt](https://github.com/eric612/MobileNet-YOLO/files/3829786/yolov3-train.prototxt.txt)

**Comment by eric612** [comment]
1. The images number of voc test set was 4952 , 
you need modify test_iter to 4952 in solver.prototxt. 

2. The training do not convergence , maybe try another solver parameter

**Comment by Rahn80643** [comment]
Thank you for your quick reply, I forgot to modiy test_iter in solver.prototxt, now darknet53-yolov3 can be trained.

I modified some hyperparameters in solver.prototxt, hope the network will converge.

**Comment by KelvinHuang666** [comment]
good job,bro.I can train my model with my own datasetuse by using your prototxt,but didn't work with author's.
